common/achievement/create.ts

```diff
@@ -31,9 +31,9 @@ async function getOrCreateUserAchievement<ID extends ActionID>(
         template.templateFor === achievement_template_for_enum.Global ||
         template.templateFor === achievement_template_for_enum.Global_Courses ||
         template.templateFor === achievement_template_for_enum.Global_Matches;
-    const existingUserAchievement = await findUserAchievement(template.id, userId, !isGlobal ? context : undefined);
+    const existingUserAchievement = await findUserAchievement(template.id, userId, !isGlobal ? context : Prisma.JsonNull);
     if (!existingUserAchievement) {
-        return await createAchievement(template, userId, context);
+        return await createAchievement(template, userId, !isGlobal ? context : Prisma.JsonNull);
```

Comment:
I'm very curious to get another explanation for `Prisma.JsonNull` here...

Reply:
The null value broke the app. But we relaced it with an empty object now.

Reply:
It's so sad that our TS configuration is in a bad shape.
The compiler could complain about many of these discussions already.
For example, if we always pass an empty object now, why is it necessary to have [this check](https://github.com/corona-school/backend/pull/950/files#diff-163cac4b632a76fd197bf1993d8cf4838413d9400aa04c57a94ed231d0c746d6R89)? It will always evaluate to true.

> The null value broke the app

The other question is, what broke exactly?

Reply:
relation is not nullable.